spec/services/github/contributions/wrappers/pull_request_spec.rb

```diff
@@ -30,4 +35,15 @@
   describe '#created_at' do
     it { expect(subject.created_at).to eq '2022-01-01' }
   end
+
+  describe '#pr_state' do
```

Comment:
Could we have a context for open PRs as well? In order to cover all the possibilities.

Reply:
Ok!